### ISSUE TITLE
Where is the label for misinformation?

### ISSUE THREAD

**Issue by Peter-Fy** [abellogin/RecSysMisInfo#1]
I noticed that there is no misinformation label in the data you provided. For example, in the file ``merged__user_item__no_filter.dat``, my understanding is that the three columns represent ``user_id``, ``item_id``, and ``is_posted_or_retweeted``, but I don’t see a column indicating whether the item is classified as misinformation. How can I obtain the misinformation label?

**Comment by abellogin** [comment]
Please, check the file `generated_dataset/userid_tweetid_claimid.csv`, where you will see which tweets have an associated claim, which refers to those that are misinformation.

**Comment by Peter-Fy** [comment]
Thank you for your answer！ I have another question, is there any way to know the time order in which the interactions occurred? It would be very helpful if I could know the chronological order of the interactions so that I can divide the training set and test set by time and try out some algorithms for sequence recommendation.

**Comment by abellogin** [comment]
I am sorry but at the moment the data does not include that information.
It could be obtained, however, by checking the metadata associated to the tweet, since the tweet id is provided.
